packages/ai/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "private": true,
+  "name": "@webstudio-is/ai",
+  "version": "0.1.0",
+  "description": "Webstudio AI Tools",
+  "author": "Webstudio <github@webstudio.is>",
+  "homepage": "https://webstudio.is",
+  "type": "module",
+  "scripts": {
+    "typecheck": "tsc --noEmit --emitDeclarationOnly false",
+    "checks": "pnpm typecheck && pnpm lint",
+    "dev": "build-package --watch",
+    "build": "pnpm build:prompts && build-package",
+    "build:prompts": "tsx ./bin/build-prompts.ts \"./src/chains/**/*.prompt.md\"",
+    "dts": "tsc --declarationDir lib/types",
+    "lint": "eslint ./src --ext .ts,.tsx --max-warnings 0",
+    "test": "NODE_OPTIONS=--experimental-vm-modules jest"
+  },
+  "devDependencies": {
+    "@types/fs-extra": "^11.0.1",
+    "@webstudio-is/jest-config": "workspace:^",
+    "@webstudio-is/scripts": "workspace:^",
+    "@webstudio-is/tsconfig": "workspace:^",
+    "fast-glob": "^3.2.12",
+    "fs-extra": "^10.1.0",
+    "jest": "^29.3.1",
+    "tsx": "^3.12.6",
+    "typescript": "5.1.3"
```

Comment:
Since these are installed in the root you can not install them in the package. All commands are available from the root.